Assets/Plugins/Source/Editor/Utility/Steam/Steamworks_Utility.cs

```diff
@@ -28,6 +28,7 @@
 
 #if !DISABLESTEAMWORKS
 using Steamworks;
+using System.Collections.Generic;
```

Comment:
Because `GetSteamInterfacesVersions()` returns a `List`, this namespace needs to be outside of !DISABLESTEAMWORKS scope

Reply:
Good catch! Changed to be outside this directive.